source/css/_common/components/sidebar/sidebar.styl

```diff
@@ -36,7 +37,7 @@
 @import "site-state";
 @import "sidebar-feed-link";
 @import "sidebar-author-links";
-@import "sidebar-blogroll.styl";
+@import "sidebar-blogroll";
 @import "sidebar-nav";
 @import "sidebar-toc";
-
+@import "sidebar-dimmer";
```

Comment:
Need to add conditional import there:
@import "sidebar-dimmer" if hexo-config('onmobile');